hypervisor/src/mshv/mod.rs

```diff
@@ -1286,4 +1286,25 @@ impl vm::Vm for MshvVm {
             )
             .map_err(|e| vm::HypervisorVmError::InitializeSevSnp(e.into()))
     }
+    /// Create AP threads for SEV-SNP enabled VM
+    #[cfg(feature = "sev_snp")]
+    fn sev_snp_ap_create(&self, apic_id: u64, vmsa_gpa: u64) -> vm::Result<()> {
+        let req = mshv_sev_snp_ap_create {
+            vp_id: apic_id,
+            vmsa_gpa,
+        };
+
+        self.fd
+            .sev_snp_ap_create(&req)
+            .map_err(|e| vm::HypervisorVmError::SevSnpApCreate(e.into()))
+    }
+    /// Issue guest request for SEV-SNP enabled guest
```

Comment:
Can we make the comment consisting SEV-SNP enabled Guest/VM. See above.

Reply:
Fixed in the new revision